.gitignore

```diff
@@ -57,3 +57,5 @@ docs/screenshots/take_all.log
 
 # ViM tags
 /tags
+
+config.py
```

Comment:
You shouldn't need to edit our `.gitignore` file. You should keep your config file out of the cloned repository.

Reply:
I'm sorry , I should have made a new branch and commited there. Am slowly learning through trial and error. I will delete this and try again. Thanks!